src/components/ButtonWithLoader.js

```diff
@@ -1,48 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import {
-    ActivityIndicator, Text, TouchableOpacity,
-} from 'react-native';
-import styles from '../styles/styles';
-import themeColors from '../styles/themes/default';
-
-const propTypes = {
-    /** The text for the button label */
-    text: PropTypes.string.isRequired,
-
-    /** Indicates whether the button should be disabled and in the loading state */
-    isLoading: PropTypes.bool,
-
-    /** Indicates whether the button should be disabled */
-    isDisabled: PropTypes.bool,
-
-    /** A function that is called when the button is clicked on */
-    onClick: PropTypes.func.isRequired,
-};
-const defaultProps = {
-    isLoading: false,
-    isDisabled: false,
-};
-
-const ButtonWithLoader = props => (
-    <TouchableOpacity
```

Comment:
I don't seee `TouchableOpacity` used anywhere in the new `Button.js` component (I could be missing it). Is there a `TouchableOpacity.js` file that could be deleted just like this `ButtonWithLoader.js` file?

Reply:
Ah, nope because this component is [built into `react-native`](https://reactnative.dev/docs/touchableopacity).